docs/html/operators.html

```diff
@@ -57,12 +57,12 @@ <h3> Relational Operators </h3>
 				<tr>
 					<td>Operator</td><td>Description</td>
 				</tr>
-				<tr> <td>==</td><td>...</td> </tr>
-				<tr> <td>!=</td><td>...</td> </tr>
-				<tr> <td>&gt;</td><td>...</td> </tr>
-				<tr> <td>&lt;</td><td>...</td> </tr>
-				<tr> <td>&gt;=</td><td>...</td> </tr>
-				<tr> <td>&lt;=</td><td>...</td> </tr>
+				<tr> <td>==</td><td>relational equals</td> </tr>
+				<tr> <td>!=</td><td>relational inequality</td> </tr>
```

Comment:
I suppose the explanation for inequality could be better.

Enhancement?

Reply:
yes, i got a bit lazy there  😃 
thanks